zabbix-mysql-dump

```diff
@@ -214,7 +214,7 @@ elementIn () {
 #
 # CHECKS
 #
-if [ ! -x /usr/bin/mysqldump ]; then
+if [ ! -x `which mysqldump` ]; then
```

Comment:
I suggest using `$(which mysqldump)`, would make it even more portable and more consistent with the rest of the file.